projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

```diff
@@ -19,21 +19,41 @@ async function hasPriorityLabels( octokit, owner, repo, number ) {
 }
 
 /**
- * Find specific plugin impacted by issue, based off issue contents.
+ * Find list of plugins impacted by issue, based off issue contents.
  *
  * @param {string} body - The issue content.
- * @returns {string} Plugin concerned by issue.
+ * @returns {Array} Plugins concerned by issue.
  */
-function findPlugin( body ) {
-	const regex = /###\sImpacted\splugin\n\n(\w*)\n/gm;
+function findPlugins( body ) {
+	const regex = /###\sImpacted\splugin\n\n([a-zA-Z ,]*)\n\n/gm;
 
 	let match;
 	while ( ( match = regex.exec( body ) ) ) {
-		const [ , plugin ] = match;
-		return plugin;
+		const [ , plugins ] = match;
+		return plugins.split( ', ' );
 	}
 
-	return null;
+	debug( `triage-new-issues: No plugin indicators found.` );
+	return [];
+}
+
+/**
+ * Find platform info, based off issue contents.
+ *
+ * @param {string} body - The issue content.
+ * @returns {Array} Platforms impacted by issue.
+ */
+function findPlatforms( body ) {
+	const regex = /###\sPlatform\s\(Simple,\sAtomic,\sor\sboth\?\)\n\n([a-zA-Z ,-]*)\n\n/gm;
+
+	let match;
+	while ( ( match = regex.exec( body ) ) ) {
+		const [ , platforms ] = match;
+		return platforms.split( ', ' ).filter( plugin => plugin !== 'Self-hosted' );
```

Comment:
Seems odd to me to filter out "Self-hosted" here (rather than below where you add labels), but if you are you should name the variable sensibly.
```suggestion
		return platforms.split( ', ' ).filter( platform => platform !== 'Self-hosted' );
```
You might also want to filter empty-string here, as above.

Reply:
Done in b378d1ce614f2159f1da8c1e841be23f16c29c48